pkg/vpath/helpers.go

```diff
@@ -0,0 +1,38 @@
+package vpath
```

Comment:
Just wonder why the package is called `vpath`?

Reply:
Temporary name. Sorta like "virtual path"? I'll probably rename/move this into some other package before I'm done.